newIDE/app/src/UI/EditorMosaic/style.css

```diff
@@ -41,3 +41,8 @@
 .mosaic-gd-theme .draggable .mosaic-window-title {
   cursor: move;
 }
+
+.mosaic-split.-column {
```

Comment:
I'm not sure if it's a good idea to add these margins: are you sure this is not breaking anything when drag'n'dropping things? 4px seems also quite large

Reply:
The 4 pixels are just for beauty, it does not affect the grip.
I added a bright color for the example, see with and without the 4pixels margin.

![image](https://user-images.githubusercontent.com/1670670/59977153-08813d80-95ce-11e9-8831-8ae84b3818db.png)

Reply:
This margins are only on horizontal separator, it's not apply on vertical ;)